### PR TITLE
heroku v3.31.2

### PR DESCRIPTION
**Pull request by dickeyxxx**

### CODE REVIEW

**Comment by mikemcquaid** [comment]
This needs a Ruby 1.9 requirement. See e.g. `mkvtoolnix.rb`

**Comment by dickeyxxx** [comment]
fixed

File: Library/Formula/heroku-toolbelt.rb [hunk 1]
```diff
@@ -1,9 +1,32 @@
...
+    next unless version
+    Version.new(version.to_s) >= Version.new("1.9")
+  end
+
+  env do
+    ENV.prepend_path "PATH", which("ruby").dirname
+  end
+
+  def message; <<-EOS.undent
+    The mkvtoolnix buildsystem needs Ruby >=1.9
```
Comment (dunn) [inline Library/Formula/heroku-toolbelt.rb]: you'll want to remove the reference to `mkvtoolnix`

Comment (dickeyxxx) [inline Library/Formula/heroku-toolbelt.rb]: fixed

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `heroku 3.31.2`.

**Pull request closed without merging**